testing/go/alter_table_test.go

```diff
@@ -197,3 +197,124 @@ func TestAlterTableRenameColumn(t *testing.T) {
 		},
 	})
 }
+
+func TestAlterTableSetColumnDefault(t *testing.T) {
```

Comment:
Just to reduce the number of individual test functions in the file, we could add these all to a larger `TestAlterTable` function.

Test scripts have a `Focus` parameter that you can set. By default it's `false` for every test, and when all are `false`, no specific tests are "focused", so they all run. When at least one is set to `true`, then _only_ that test is run. You can set multiple to `true` to run only the specific ones. There's also a CI check to make sure that you don't accidentally commit a focused test, since it's only for local testing. I'm assuming that's why there are individual test functions at least, so sort of implement the behavior of allowing specific tests to run.